client/app/common/less/popover.less

```diff
@@ -55,8 +55,9 @@
 
                 .back-btn {
                     position: absolute;
-                    left: 10px;
+                    left: 12px;
                     top: -3px;
+                    font-size: 20px;
```

Comment:
Please use relative values and ui-kit less variables ;)

Reply:
popover.less uses a lots of pixel values, so ... no
the whole thing should be rewritten cleany but it will come later ;)